src/views/App/index.tsx

```diff
@@ -50,6 +50,7 @@ const AppRoutes = () => {
         <SecureRoute
           path="/governance-task-list/:systemId"
           exact
+          render={({ component }: any) => component()}
```

Comment:
We need this for memory leak issues with `SecureRoute` unmounting components. This fixes the console error.